tests/sqlite/macros.rs

```diff
@@ -16,6 +16,27 @@ async fn macro_select() -> anyhow::Result<()> {
     Ok(())
 }
 
+macro_rules! gen_macro_select_concats {
+    ($param:literal) => {
+        #[sqlx_macros::test]
+        async fn macro_select_concat_single() -> anyhow::Result<()> {
+            let mut conn = new::<Sqlite>().await?;
+
+            let account = sqlx::query!("select " + $param + " from accounts where id = 1")
+                .fetch_one(&mut conn)
+                .await?;
+
+            assert_eq!(1, account.id);
+            assert_eq!("Herp Derpinson", account.name);
+            assert_eq!(account.is_active, Some(true));
+
+            Ok(())
+        }
+    }
```

Comment:
Looks like rustfmt wants a semicolon after the brace here: https://github.com/launchbadge/sqlx/pull/474/checks?check_run_id=835719828

Reply:
Should be ok now hopefully